.github/workflows/CI.yml

```diff
@@ -523,6 +523,10 @@ jobs:
         if: steps.e2etest-release.outputs.cache-hit != 'true'
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
+        with:
```

Comment:
I don't think this action takes the credentials. https://github.com/aws-actions/amazon-ecr-login#credentials-and-region
That's done by `aws-actions/configure-aws-credentials@v1`, which happens in an earlier step.

Reply:
True. I think you are using this instead of aws-actions/amazon-ecr-login@1 aws-actions/amazon-ecr-login@v1